### ISSUE TITLE
Incorrect tool-tip and aria-label are defined for the Source button.

### ISSUE THREAD

**Issue by azaslonov** [paperbits/paperbits-demo#133]
User Experience:

If, the tool-tip and aria-label are incorrectly defined for any control, then the user who depends upon Screen Reader for navigation might get affected hence failed to get the correct information of the control.

Note: User credentials should NOT be included in the bug.

 

Repro steps:

1. Open Azure portal with valid credentials and create a API Management.
2. Click Developer Portal button to navigate to the portal.
3. Wait portal to load in design mode
4. Navigate to the Left nav and select Pages button.
5. Navigate to the Pages window. Select "APIs: List" option.
6. Navigate to the any editable control details and select Add widget.
7. Navigate to the Video player and select it.
8. Navigate to the various controls of the "Video player" dialog box.
9. Navigate to source button.
10. Open developer toolbar via pressing F12 key and check the tool-tip and aria-label of the button.
 

Actual Result:

Incorrect tool-tip and aria-label are defined for the collapsed (+), expand(-) icons buttons i.e toggle section, Screen Reader announce “Toggle section” button collapse.
 

Expected Result:

Correct tool-tip and aria- label should be defined for all ‘collapsed (+), expand(-) icons’ buttons as ‘source’ expand/collapsed button and title=Source.

Note:

1. Similar issue exist with Chrome+JAWS+NVDA.
2. Similar issue exist throughout the application for collapsed(+) expand(-) icons button available on the dialog box.
 

MAS Reference:

MAS 4.1.2 - Name, Role, Value (40B)

 

Reference Links

Accessibility Insights! - Identify accessibility bugs before check-in and make bug fixing faster and easier.
External Bug Process: If this bug belongs to external team, mark it as resolved/done and assign it      back to the tester with notes on where to file/route the bug. For more information please use this Link. “HCL Staff should not log any third party external bugs. Should be routed to edad team”.
Please reach out to C&AI Teams channel for any process related queries.

![image](https://user-images.githubusercontent.com/2320302/79024908-2dcd4100-7b39-11ea-8825-51cca281ef32.png)

**Comment by azaslonov** [comment]
Fixed.
